flow/Knobs.cpp

```diff
@@ -90,6 +90,14 @@ void FlowKnobs::initialize(Randomize randomize, IsSimulated isSimulated) {
 	init( TRACING_UDP_LISTENER_ADDR,                   "127.0.0.1"); // Only applicable if TracerType is set to a network option
 	init( TRACING_UDP_LISTENER_PORT,                          8889 ); // Only applicable if TracerType is set to a network option
 
+	// Native metrics
+	init( METRICS_DATA_MODEL, 							  "otel", Atomic::NO ); // Defines the data model for which metrics are emitted
```

Comment:
Although this should be `none` by default, it should also be buggified to randomly be one of the other options, so the export functionality is tested in simulation.